package-lock.json

```diff
@@ -7731,29 +8923,22 @@
       }
     },
     "react-select": {
-      "version": "1.0.0-rc.10",
-      "resolved": "https://registry.npmjs.org/react-select/-/react-select-1.0.0-rc.10.tgz",
-      "integrity": "sha1-8Tc0YlD5JVyXn7+iGGCJmSh3I1A=",
+      "version": "1.0.1",
```

Comment:
I think this changed the behaviour of the dropdown menus on Notes from Nature the last time we accidentally upgraded react-select. I thought the package file had been locked to a version of 1.0.0.